packages/veritone-react-common/src/components/MediaInfoPanel/index.js

```diff
@@ -360,20 +321,19 @@ class MediaInfoPanel extends Component {
                 Program Live Image
                 {get(
                   this.props.tdo,
-                  'details.veritoneProgram.signedProgramLiveImage.length',
+                  'details.veritoneProgram.programLiveImage.length',
```

Comment:
That's ok as long as this uri is signed. The thing is that currently stored uri here are not signed.